definition_tooling/validator/cli.py

```diff
@@ -0,0 +1,65 @@
+import sys
+import traceback
+from contextlib import contextmanager
+from pathlib import Path
+
+from rich import print
+from typer import Argument, Typer
+
+from definition_tooling.log import print_error, print_success, print_table
+from definition_tooling.validator.core import DefinitionValidator
+from definition_tooling.validator.errors import ValidatorError
+
+cli = Typer()
+
+
+def print_dashes(char="=", length=79):
+    print(char * length)
+
+
+@contextmanager
+def header():
+    print_dashes()
+    yield
+    print_dashes()
+
+
+@cli.command(help="Validate Data Product definitions")
+def cli_validate_specs(
+    path: str = Argument(..., help="Path to directory with definitions in JSON format")
+):
+    sys.exit(validate_specs(Path(path)))
+
+
+def validate_specs(path: Path) -> int:
+    with header():
+        print(f"OpenAPI specs root path: {path}")
+
+    passed, failed = 0, 0
+    for spec_path in path.glob("**/*.json"):
+        print(f"File: {spec_path}")
+        try:
+            DefinitionValidator(spec_path).validate()
+        except Exception as exc:
+            if isinstance(exc, ValidatorError):
+                print(f"{exc.__class__.__name__}: {str(exc)}")
```

Comment:
Some of the errors are raised like `raise err.PostMethodIsMissing` and don't have any custom message. I belive in those case you will be printing just `raise err.PostMethodIsMissing: ` which might look a bit weird. Maybe check I didn't miss anything (didn't run it, just assume based on code) and maybe improve it to skip the colon and space if there's no message.